tss/node/cmd/tssnode/cmd.go

```diff
@@ -12,7 +12,6 @@ func Command() *cobra.Command {
 	}
```

Comment:
command `tssnode` => `node`, as we have parent command named `tss`

Reply:
solved